src/main/java/esme/Ui.java

```diff
@@ -0,0 +1,163 @@
+package esme;
+
+import esme.task.TaskList;
+
+public class Ui {
+    private TaskList taskList;
+    private static final String SEPARATOR = "--------------------------------------------------------------------------------------------------";
+
+    public Ui() {
+        taskList = new TaskList();
+    }
+
+    public void addTaskToList(String command, String input) {
+        String description;
+        switch(command) {
+        case "todo":
+            description = taskList.addTodoTask(input);
+            break;
+        case "deadline":
+            description = taskList.addDeadlineTask(input);
+            break;
+        case "event":
+            description = taskList.addEventTask(input);
+            break;
+        default:
+            System.out.println("Invalid command");  // Check again
+            return;
+        }
+        displayLine(true);
+        System.out.println("\tThe stars have aligned and " + description + " is now part of your destiny!");
+        displayLine(true);
+        System.out.println("\tTime to work! You got " + taskList.getNumberOfTasks() + " tasks waiting for you!");
+        displayLine(true);
+    }
+
+    /**
+     * Checks if the given task index is valid in terms of the task list.
+     * 
+     * @param index The index to check.
+     * @return True if the index is valid, false otherwise.
+     */
```

Comment:
I like how you write comments targeting other programmers reading the code to explain what the functions does